lib/purchases_flutter.dart

```diff
@@ -1230,6 +1230,16 @@ class Purchases {
     final response = Map<String, dynamic>.from(result!);
     return response;
   }
+
+  /// Web-only: Loads and caches some optional data in the Purchases SDK.
+  /// Currently only fetches branding information.
+  ///
+  /// You can call this method after configuring the SDK to speed up the first call to purchase.
+  /// This method has no effect on non-web platforms.
+  static Future<void> preload() async {
+    if (!kIsWeb) return;
+    await _channel.invokeMethod('preload');
+  }
```

Comment:
I'm going to remove this for now, and add it later if needed... Seems like a very generic API that could be confused with it doing something else. We can add it back in the future if needed.